comp/src/main/java/frc/robot/subsystems/Elevator/ElevatorSubsystem.java

```diff
@@ -153,13 +153,49 @@ public Command stopElevator() { // for manual control, sick
         });
     }
 
-    public Command descoreAlgae(double setpoint) {
+    public Command scoreAlgae() {
+        return Commands.sequence(goToSetpoint(Constants.SetpointConstants.REEF_L4),
+                Commands.waitUntil(() -> {
+                    return isAtSetpoint();
+                }),
+                arm.goToSetpoint(Constants.ArmConstants.ALGAE_SCORE));
+        // intake.ejectAlgae());
+    }
+
+    // public Command descoreAlgae(double setpoint) {
+    // return Commands.sequence(goToSetpoint(setpoint),
+    // Commands.waitUntil(() -> {
+    // return isAtSetpoint();
+    // }),
+    // arm.goToSetpoint(Constants.ArmConstants.ALGAE_DESCORE),
+    // intake.intakeJustOut());
+    // }
+
+    public Command intakeAlgae(double setpoint) {
+        return Commands.sequence(goToSetpoint(setpoint),
+                Commands.waitUntil(() -> {
+                    return isAtSetpoint();
+                }),
+                arm.goToSetpoint(Constants.ArmConstants.ALGAE_INTAKE),
```

Comment:
This won't do anything because `Constants.ArmConstants.ALGAE_INTAKE` is outside of the min/max.